### ISSUE TITLE
Check authentication methods for email services

### ISSUE THREAD

**Issue by hannob** [hardenize/hardenize-public#45]
It would be nice if hardenize would show supported authentication methods for SMTP/IMAP/POP3, e.g. whether SCRAM is supported.

In a further step one could think about good recommendations for secure selections, but a first step would be to just display them.

**Comment by ivanr** [comment]
Yes, I agree, but checking that would make sense for SMTP running on submission ports, not port 25. We're planning to test all public services. The only thing holding us back at this point is that we need to redesign the report to stay reasonably small while we expand to cover more services.
